### ISSUE TITLE
Cannot store secrets via API, no such object path /org/freedesktop/secrets/aliases/default

### ISSUE THREAD

**Issue by haslersn** [keepassxreboot/keepassxc#9816]
I set up keepassxc as my secret service daemon and configured git to use git-credential-libsecret. However, git fails to store secrets with the following error. _EDIT: Other applications also fail to store secrets with the same error._

```
** (process:690063): CRITICAL **: 12:41:16.182: store failed: No such object path '/org/freedesktop/secrets/aliases/default'
```

I do have a database open with an exposed database group.

_EDIT: This issue seems to be kind of intermittent. On some starts of KeePassXC it works as expected._

## Steps to Reproduce
[NOTE]: # ( Provide a simple set of steps to reproduce this bug. )
1. Enable secret service integration in KeePassXC
2. Open KeePassXC database and (if not done yet) configure an exposed database group
3. Try to store a password, e.g. using `git-credential-libsecret`:

```console
$ echo -e "protocol=https\nhost=example.com\nusername=bob\npassword=secr3t" | git-credential-libsecret store

** (process:690719): CRITICAL **: 12:48:51.627: store failed: No such object path '/org/freedesktop/secrets/aliases/default'
```

## Context
[NOTE]: # ( Give us any additional information you may have. )

KeePassXC - Version 2.7.4

Qt 5.15.10
Debugging mode is disabled.

Operating system: NixOS 23.05 (Stoat)
CPU architecture: x86_64
Kernel: linux 6.1.45

Enabled extensions:
- Auto-Type
- Browser Integration
- SSH Agent
- KeeShare
- YubiKey
- Secret Service Integration

Cryptographic libraries:
- Botan 2.19.3

**Comment by michaelk83** [comment]
Probably need to expose a database group. Check the settings.

**Comment by haslersn** [comment]
@michaelk83 Did you read the issue description?

> I do have a database open with an exposed database group. Other applications such as network manager can successfully store secrets

**Comment by droidmonkey** [comment]
> Other applications such as network manager can successfully store secrets

Please file a bug with git-credential-libsecret, they are not following the standard, technically. They are supposed to query for available stores and choose one explicitly. 

There was some other chatter about this but seems unrelated: https://github.com/keepassxreboot/keepassxc/issues/7832

Also this https://github.com/keepassxreboot/keepassxc/issues/8611

**Comment by haslersn** [comment]
I was wrong about this issue being specific to git-credential-libsecret. The issue is kind of intermittent, which is probably why at first I only noticed it with git-credential-libsecret.

Since then, I have also noticed that error when

- connecting to a new WiFi with network manager
- using `secret-tool store` (from libsecret)
- running the python snipped
  ```python3
  import secretstorage
  conn = secretstorage.dbus_init()
  collection = secretstorage.get_default_collection(conn)
  ```
- running
  ```console
  dbus-send --session --type=method_call --print-reply \
    --dest=org.freedesktop.secrets /org/freedesktop/secrets/aliases/default \
    org.freedesktop.DBus.Introspectable.Introspect
  ```

I was able to resolve the issue for now by restarting KeePassXC, which crashed the desktop environment (Gnome), logging back in again and starting KeePassXC again.
